### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by Edelstadt**
Adding a few simple tests, use of pytest and travis. Please, do not kill me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+[![Build Status](https://travis-ci.org/Edelstadt/czech-holidays.svg?branch=patch-2)](https://travis-ci.org/Edelstadt/czech-holidays)
```
Comment (honzajavorek) [inline README.md]: please leave the README as `.rst`, badges can be added also in rst, like this:

```rst
.. image:: https://travis-ci.org/honzajavorek/czech-holidays.svg
   :target: https://travis-ci.org/honzajavorek/czech-holidays
```

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+pytest
+python-dateutil
```
Comment (honzajavorek) [inline requirements.txt]: - python-dateutil is already in setup.py as dependency of czech-holidays
- pytest is needed for testing, so I'd either use tox or directly written `pip install pytest` to `.travis.yml`

I don't want to maintain two lists of dependencies.

File: holidays_test.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# Range of years, for example
+FIRST_YEAR = 2016
+LAST_YEAR = 2030
+
+
+def test():
+    index = 0
+    for year in range(FIRST_YEAR, LAST_YEAR):
+        holidays = Holidays(year)
+#  first holiday is always 1 January
```
Comment (honzajavorek) [inline holidays_test.py]: please indent the comment

File: holidays_test.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        assert str(holidays[0]) == str(year) + "-01-01"
+#  Good Friday tests (controlling movable holidays)
```
Comment (honzajavorek) [inline holidays_test.py]: dtto

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+language: python
+python:
+    - "2.7"
```
Comment (honzajavorek) [inline .travis.yml]: I wonder if czech-holidays work also for 3.x?

**Comment by honzajavorek** [comment]
No killing, I promise! Thanks! Just address my comments, please :)

File: .travis.yml (hunk 5 above)
Comment (Edelstadt) [inline .travis.yml]: Probably not. Not simply (dateutil)

Comment (honzajavorek) [inline .travis.yml]: ok, shame on me! :)

**Comment by honzajavorek** [comment]
https://github.com/honzajavorek/czech-holidays/pull/10 supersedes this.

**Pull request closed without merging**